docs/manual/common/guide/production/LightbendPlatform.md

```diff
@@ -0,0 +1,44 @@
+# Using Lightbend Platform with Lagom
+
+Lagom is part of the [Lightbend Platform](https://www.lightbend.com/lightbend-platform) — an operationally robust solution for deploying Reactive Microservices, adding the ability to infuse your applications with intelligence based on real-time streaming and Machine Learning (ML). Lagom services can be developed, deployed and run in production using only open source components, however we recommend considering the advanced features provided by Lightbend Platform.
+
+This page highlights the Lightbend Platform features that are especially useful for most Lagom users, but there are many more. You can find an overview on the [Lightbend web site](https://www.lightbend.com/lightbend-platform) or [contact Lightbend](https://www.lightbend.com/contact) to learn more.
+
+## Akka Commercial Addons
+
+Lagom is based on Akka, a set of open-source libraries for designing scalable, resilient systems that span processor cores and networks. [Akka Commercial Addons](https://developer.lightbend.com/docs/akka-commercial-addons/current/index.html) is a suite of useful components that complement Akka and Lagom. These can be included as dependencies of your Lagom services to enable their functionality. It includes:
```

Comment:
Naming changes, I won't repeat.

Reply:
I'll do this separately once it's changed upstream.